src/sveltekit/sdk-setup.ts

```diff
@@ -446,13 +446,16 @@ function wrapHandle(mod: ProxifiedModule<any>): void {
     } else if (declaration.type === 'VariableDeclaration') {
       const declarations = declaration.declarations;
       declarations.forEach((declaration) => {
-        // @ts-ignore - id should always have a name in this case
-        if (!declaration.id || declaration.id.name !== 'handle') {
+        if (
+          !declaration.id ||
+          declaration.id.type !== 'Identifier' ||
+          (declaration.id.name && declaration.id.name !== 'handle')
+        ) {
```

Comment:
I lied in this ts-ignore comment 😅  There's no guarantee for the declaration.id to always have a name. The new check is more defensive and makes no more assumptions.